packages/usa-tooltip/src/index.js

```diff
@@ -357,6 +359,24 @@ const setUpAttributes = (tooltipTrigger) => {
   return { tooltipBody, position, tooltipContent, wrapper };
 };
 
+const handleEscape = () => {
+  const activeTooltips = document.querySelectorAll(".is-set");
+  if ( event.keyCode === 27 ) {
```

Comment:
Note that `event.keyCode` is deprecated and should be replaced with `event.key === 'Escape'`